### PR TITLE
updating the analyses

### PR DESCRIPTION
**Pull request by lkasaitis**

### CODE REVIEW

File: beneficiary_final_demographics.ipynb
Comment (jawalonoski) [inline beneficiary_final_demographics.ipynb:1]: Does not work because `beneficiary_final.csv` does not exist. Should it be `data/beneficiary_2022.csv`?

Comment (jawalonoski) [reply beneficiary_final_demographics.ipynb:1]: When attempts to use `beneficiary_2022.csv`:

```
ValueError: Usecols do not match columns, columns expected but not found: ['BENE_SEX_IDENT_CD', 'AGE']
```

File: chronic_conditions_apriori.ipynb
Comment (jawalonoski) [inline chronic_conditions_apriori.ipynb:1]: Again `beneficiary_final.csv` is not one of the files.

File: synthetic_readmissions_sdoh.ipynb
Comment (jawalonoski) [inline synthetic_readmissions_sdoh.ipynb:1]: Replacing `beneficiary_final.csv` with `beneficiary_2022.csv` does not work -- column name error:

```
ValueError: Usecols do not match columns, columns expected but not found: ['BENE_SEX_IDENT_CD', 'FIPS_STATE_CNTY_JAN_CD', 'AGE']
```

File: synthetic_readmissions_sdoh_+_race.ipynb
Comment (jawalonoski) [inline synthetic_readmissions_sdoh_+_race.ipynb:1]: Replacing beneficiary_final.csv with beneficiary_2022.csv does not work -- column name error:

```
ValueError: Usecols do not match columns, columns expected but not found: ['BENE_SEX_IDENT_CD', 'FIPS_STATE_CNTY_JAN_CD', 'AGE']
```

File: unique_claims_per_beneficiary.ipynb
Comment (jawalonoski) [inline unique_claims_per_beneficiary.ipynb:1]: Error on reading `pde.csv`
```
ValueError: Usecols do not match columns, columns expected but not found: ['CLM_GRP_ID']
```

**Review by jawalonoski — CHANGES_REQUESTED** [review]
There are a couple of errors that need to be corrected, especially references to `beneficiary_final.csv` which does not exist after conversion from BFD to CCW format.

File: synthetic_feature_selection_input.ipynb
Comment (jawalonoski) [inline synthetic_feature_selection_input.ipynb:1]: Running this notebook yields:
```
ValueError: Usecols do not match columns, columns expected but not found: ['BENE_ENTLMT_RSN_CURR']
```

The correct column should be `ENTLMT_RSN_CURR`.

File: synthetic_sdoh_feature_selection_limit_feature_class.ipynb
Comment (jawalonoski) [inline synthetic_sdoh_feature_selection_limit_feature_class.ipynb:1]: I was unable to run this notebook successfully, as `synthetic_feature_selection_input.ipynb` did not complete successfully.

**Review by jawalonoski — CHANGES_REQUESTED** [review]
Some of my previous comments seem to be related to my repo being out of date.

I encountered an additional two issues...

File: synthetic_feature_selection_input.ipynb
Comment (lkasaitis) [reply synthetic_feature_selection_input.ipynb:1]: Corrected column to ```ENTLMT_RSN_CURR```.

File: synthetic_sdoh_feature_selection_limit_feature_class.ipynb
Comment (lkasaitis) [reply synthetic_sdoh_feature_selection_limit_feature_class.ipynb:1]: reran after correcting ```synthetic_feature_selection_input.ipynb```

File: synthetic_feature_selection_input.ipynb
Comment (jawalonoski) [reply synthetic_feature_selection_input.ipynb:1]: On saving the data frame as Excel, I now get

```
ModuleNotFoundError: No module named 'xlsxwriter'
```

Either the notebook is missing an `import`, or we need to update the `requirements.txt` file, or both.

Comment (lkasaitis) [reply synthetic_feature_selection_input.ipynb:1]: added ```import xlsxwriter``` to ```synthetic_feature_selection_input.ipynb``` and updated the ```requirements.txt``` file.

**Review by jawalonoski — APPROVED** [review]

**Pull request merged**